### ISSUE TITLE
[request] Please add a scheduler to Flood

### ISSUE THREAD

**Issue by binhex** [jfurrow/flood#180]
So unless i'm missing something then it looks like Flood doesn't currently have any way of controlling UL/DL via a hourly schedule, so this is a feature request to have this added in.

This is something that people who are not on completely unlimited connections will find necessary to prevent high dl costs and/or blocking/choking of their connection. It is also handy to be able to restrict bandwidth at set times when you need to perform other bandwidth intensive tasks.

Just to be clear i know Flood is new code and so i completely understand that not everything is in place yet, but i would love to see this added in.

And lastly, thanks for creating Flood.

**Comment by jfurrow** [comment]
Agreed, this is needed! I've always planned on it, so it will come in due time. :)
